lib/jsonwp-proxy/proxy.js

```diff
@@ -72,7 +72,7 @@ class JWProxy {
       throw new Error(`Did not know what to do with url '${url}'`);
     }
 
-    const stripPrefixRe = new RegExp('^.+(/(session|status).*)$');
+    const stripPrefixRe = new RegExp(`^${this.base}(/(session|status).*)$`);
```

Comment:
shall we escape `this.base` before inserting it into the pattern?

Reply:
We could, if we don't trust driver writers.